modules/apps/headless/headless-content-common/bnd.bnd

```diff
@@ -0,0 +1,7 @@
+Bundle-Name: Liferay Headless Content Common
```

Comment:
I think it would be a good idea to create the headless content common module in a separate commit and then refactor to use it.